Source/WTF/wtf/threads/Signals.h

```diff
@@ -56,8 +59,16 @@ enum class Signal {
     AccessFault, // For posix this is both SIGSEGV and SIGBUS
     NumberOfSignals = AccessFault + 2, // AccessFault is really two signals.
     Unknown = NumberOfSignals
+#else
+    FloatingPoint,
+    IllegalInstruction,
+    AccessFault,
+    NumberOfSignals = AccessFault + 1,
+    Unknown
+#endif
```

Comment:
@Constellation do you want these shared as well? I would say yes.

Reply:
Yeah, that's nice if we can share them. But there are a bit of hacks like `NumberOfSignals = AccessFault + 2`, so it is OK to keep them separate.